src/main/model/CurlCommand.java

```diff
@@ -0,0 +1,95 @@
+package main.model;
+/* Model for the curl command: \"curl -X 'GET' \
+       'https://financial-integration.service.cnqr.tech/fi/v1/companies/1/transactiontypes/expense/transactions?limit=0&docId=%s&ignoreDocumentStatus=yes' \
+       -H 'accept: application/json' \
+       -H 'documentFormatAs: json' \
+      -H 'Authorization: Bearer $JWT'\" \
+       > ~/Downloads/%s.json;
+*/
+public class CurlCommand {
+    private String method;
+    private String url;
+    private String ignoreDocStatus;
+    private String  accept;
```

Comment:
While this is what the command says, it's not really a descriptive name. This is actually a header called content-type. I'd name it something referencing both of those things.